sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -133,9 +134,8 @@ abstract class StreamExecution(
             .stripMargin)
       }
     }
-    val checkpointDir = checkpointPath.makeQualified(fs.getUri, fs.getWorkingDirectory)
```

Comment:
`makeQualified` disappeared from here, right?

Reply:
I think so. We're using the `checkpointFileManager` for this behavior now in the PR so we are not directly making FS operations. Looking inside of the default FileSystem impl we don't explicitly translate this to a fully-qualified path, do you think that would be an issue?

Reply:
Since this ends-up in behavior change it may break existing queries. I think there must be a reason when we do such change.